lib/src/index.ts

```diff
@@ -74,10 +74,17 @@ export { default as NxTabPanel, NxTabPanelProps } from './components/NxTabs/NxTa
 
 export {
   default as NxTreeView,
-  NxTreeViewChild,
+  NxCollapsibleItemsChild as NxTreeViewChild,
   Props as NxTreeViewProps,
-  NxTreeViewChildProps
-} from './components/NxTreeView/NxTreeView';
+  NxCollapsibleItemsChildProps as NxTreeViewChildProps
+} from './components/NxCollapsibleItems/NxCollapsibleItems';
+
+export {
+  default as NxCollapsibleItems,
+  NxCollapsibleItemsChild,
```

Comment:
We only want this exposed as the `.Child` subcomponent (and as the deprecated alias)
```suggestion
```